get-started/guide-neo4j-desktop/guide-neo4j-desktop.adoc

```diff
@@ -0,0 +1,147 @@
+To remove the project, hover over the `Project` row in the projects menu and click the recycle can icon that appears on the right side (image:{img}delete_project_icon.png[width=20]).
+
+=== Settings
+
+Application settings can be modified or viewed by clicking the gear towards the bottom of the left-side menu in Neo4j for desktop (image:{img}settings_icon.png[width=20]).
+You can modify any of these settings at any time.
+
+The first block is for privacy settings where you can adjust what data is sent back to Neo4j for assistance and future improvements.
+Under `Neo4j Distributions`, it shows which versions of the database are used in your projects (if any) and lists the currently set up projects and databases.
+
+The last section on `Developer Tools` is to add and integrate graph apps with Neo4j.
+You can also access the javascript console of an application for debugging and error reporting through the developer tools.
+
+=== Profile and About
+
+The person icon (image:{img}profile_icon.png[width=20]) shows the profile you created when you signed into Desktop and allows you to sign out of the current account and into another one.
+It shows who is currently logged into Neo4j for desktop.
+
+The bottom icon (image:{img}neo4j_icon.png[width=20]) is information about Neo4j itself and includes links to the main company website, as well as release and license information.
+
+== Creating a graph database
+
+From the Projects icon on the left-side menu, choose (or create) a project.
+A panel will display on the right to show the databases in your project and the plugins available or installed.
+
+To create a database, click in the dotted square that says `New Graph`.
+Then click the kbd:[Create a Local Graph] button.
+Next, you will type in a name for your database and set a password and choose a version (default is latest version), then click kbd:[Create].
+It may take a few minutes to download the version and actually create the database.
+
+****
+[NOTE]
+Please remember your database password. 
+If you forget it, you will have to remove the credentials file in `data/dbms/authorization`.
+****
+
+You have now created your own graph database!
+Now we will walk through some tools to interact with the database and data.
+
+At the bottom of your newly created database are two buttons.
+One is to manage the database and the other is to start the database (when the database is running, the button is to stop the database).
+
+image::{img}db_section.png[]
+
+== Managing a database
+
+When you click the kbd:[Manage] button, the resulting pane opens up several functionalities.
+
+image::{img}manage_db.png[]
+
+In the top section is your database name with buttons underneath it for Start, Stop, and Restart of the database.
+The buttons next to those allow you to open the folder location of the database (for modifying data files and configuration) and to open the Neo4j Browser to interact with data in the database.
```

Comment:
also to access e.g. the import folder for placing CSV files
with the drop-down on the right side.